### PR TITLE
Update PING

### PR DESCRIPTION
**Pull request by plehegar**
* Chair affiliation (Pete)
* CoC
* Slack invite
* W3C Calendar

### CODE REVIEW

File: template/ig/privacy [hunk 1]
```diff
@@ -2,19 +2,22 @@ Welcome to the W3C Privacy Interest Group, known as PING.
...
 
-* We meet the 1st and 3rd Thursday of each month at noon US Eastern, 9AM Pacific (1600 or 1700 UTC depending on the season).
+* We also use the ping Slack channel on the w3ccommunity workspace. If you need an invite, use
+  https://www.w3.org/slack-w3ccommunity-invite
 
-    Meeting coordinates can be found at: https://www.w3.org/Privacy/IG/
-    
-    We use a shared Cryptpad for minutes, also linked above.
+* The group meets twice a month:
+  https://auth.w3.org/?url=https://www.w3.org/groups/wg/{{ group.shortname }}/calendar
```
Comment (plehegar) [inline template/ig/privacy]:
```suggestion
  https://auth.w3.org/?url=https://www.w3.org/groups/ig/{{ group.shortname }}/calendar
```

File: template/ig/privacy [hunk 2]
```diff
@@ -2,19 +2,22 @@ Welcome to the W3C Privacy Interest Group, known as PING.
 
 We are delighted to have you join us. Our mission is to improve privacy on the Web.
 
-Our co-chairs are Pete Snyder (Brave), Christine Runnegar (invited expert), and Nick Doty (CDT). Philippe Le Hegaret is our W3C contact.
+Everyone who is involved in the {{ group.name }} is expected to behave in accordance with the W3C Code of Conduct:
+  https://www.w3.org/policies/code-of-conduct/
+
+Our co-chairs are Pete Snyder (Invited Expert), Christine Runnegar (invited expert), and Nick Doty (CDT). Philippe Le Hegaret is our W3C contact.
```
Comment (samuelweiler) [inline template/ig/privacy]: for consistency, s/Invited Expert/invited expert/

Comment (plehegar) [inline template/ig/privacy]:
```suggestion
Our co-chairs are Pete Snyder (invited Expert), Christine Runnegar (invited expert), and Nick Doty (CDT). Philippe Le Hegaret is our W3C contact.
```

Comment (plehegar) [reply template/ig/privacy]: fixed. merci!

**Review by caribouW3 — APPROVED** [review]

**Pull request merged**